README.md

```diff
@@ -108,3 +108,49 @@ More about how the upstream action works can be found [here](https://github.com/
 ### Add new components to Release Please config file
 
 In order for components to be released, they must be in the [release-please-config.json](./release-please-config.json) file. Always ensure new components are added to this file.
+
+Once the component is added, as part of using the latest version of an action, it's a good practice to update the README file of each shared workflow every time there is a new release.
+
+To do so, after adding the name of the new action which needs to be released in the [release-please-config.json](./release-please-config.json) file, the `"extra-files"` field should also be added and include the `README.md` like:
+
+```json
+  ...
+  "packages": {
+    "actions/my-new-action": {
+      "package-name": "my-new-action",
+      "extra-files": ["README.md"]
+    },
+  }
+```
+
+Also, the following block should be added in the README file which will be responsible for updating the version with a new one:
+
+`README.md`:
+
+````
+# my-new-action
+
+This is my new action which does awesome stuff!
+
+<!-- x-release-please-start-version --> # beginning of the release please block
```

Comment:
If people copy and paste this with the `# beginning of the release please block` text, that will show up in their README. Might suggest the following 3...

```suggestion
<!-- x-release-please-start-version -->
```

Reply:
wanted to make sure that people know what the change is really. it's clear from the docs I guess, so fine to remove it.